pyproject.toml

```diff
@@ -14,7 +14,7 @@ keywords = ["functional data analysis"]
 dependencies=[
         "Cython",
         "matplotlib",
-        "numpy",
+        "oldest-supported-numpy",
```

Comment:
I think this was fine as `numpy`. You need the oldest version to compile, but then you allow a newer one.

Reply:
Good point, I was just being super sure, we can move this one back.

Reply:
I think this would fix the runtime `numpy` version to an older one, which is not desirable.

Reply:
After researching that's correct, fixed on master